probe-rs/src/probe/jlink/mod.rs

```diff
@@ -620,6 +614,29 @@ impl DebugProbe for JLink {
     fn into_probe(self: Box<Self>) -> Box<dyn DebugProbe> {
         self
     }
+
+    fn try_get_arm_interface<'probe>(
+        mut self: Box<Self>,
+    ) -> Result<Box<dyn ArmProbeInterface + 'probe>, (Box<dyn DebugProbe>, DebugProbeError)> {
+        if self.supported_protocols.contains(&WireProtocol::Swd) {
+            if let Some(WireProtocol::Jtag) = self.protocol {
+                log::warn!("Debugging ARM chips over JTAG is not yet implemented for JLink.");
```

Comment:
I am not sure what policy we implement regarding logging now (maybe we should discuss this some time?) but shouldn't this be an `error!`?

Reply:
It's not really an error in my opinion, because there's nothing actually wrong.

The idea of this output is to inform users that the problem is a missing implementation, not an actual error.